pydis_site/templates/events/pages/code-jams/the-code-style-guide.html

```diff
@@ -0,0 +1,299 @@
+                        #
+                        # Here starts a new important paragraph.
+                </code>
+            </pre>
+        </li>
+        <li>
+            <b>Inline comments:</b> You should prefer block comments over inline comments and use inline comments only where it is really necessary.
+            Never use inline comments to explain obvious things what line do. If you want to use an inline comment on a variable, think first,
+            maybe you can use a better variable name instead. After code and before the start of inline comments should be at least two spaces.
+            Just like block comments, inline comments also have to start with <code>#</code> what should be followed by a single space.
+            <pre>
+                <code class="language-python">
+                    # Do not use inline comments to explain things
+                    # that the reader can understand even without the inline comment.
+                    my_variable = "Value!"  # Assign value to my_variable
+
+                    # Here better variable name can be used like shown in the second line.
+                    x = "Walmart"  # Shop name
+                    shop_name = "Walmart"
+
+                    # Sometimes, if something is not obvious, then inline comments are useful.
+                    # Example is from PEP 8.
+                    x = x + 1  # Compensate for border
+                </code>
+            </pre>
+        </li>
+        <li>
+            <b>Docstrings:</b> Last, but not least important comment type is docstring, which is a short version of documentation string.
+            Docstring rules haven't been defined by PEP 8, but by <a href="https://www.python.org/dev/peps/pep-0257">PEP 257</a> instead.
+            Docstrings should start and end with three quotes ("""). There are two types of docstrings: one-line docstrings and multiline docstrings.
+            One-line docstrings have to start and end in the same line, while multiline docstrings start and end in different lines.
+            Multiline docstring has two parts: summary line and a longer description, which are separated by one empty line.
+            The multiline sentence starting and ending quotes should be at different lines than the content.
+            <pre>
+                <code class="language-python">
+                    # This is a one-line docstring.
+                    """This is one line module docstring."""
+
+
+                    # This is a multiline docstring.
+                    def my_function():
+                        """
+                        This is the summary line.
+
+                        This is the description.
+                        """
+                </code>
+            </pre>
+        </li>
+    </ul>
+
+    <h2>Too much for you?</h2>
+    <p>
+        Is all this PEP 8 rules stuff making your head exploding? We have something for you! We have a song!
```

Comment:
This reads weirdly to me, maybe something like `Do all these PEP 8 rules make your head explode?` would be better

Reply:
I changed it, but used `style rules` instead of `PEP 8 rules`.